raiden/utils/gas_reserve.py

```diff
@@ -68,7 +71,10 @@ def _get_required_gas_estimate_for_state(raiden) -> int:
             token_network_registry_address=registry_address,
             token_address=token_address,
         )
-        num_opening_channels = raiden.chain.token_network(
+        if token_network_address is None:
```

Comment:
Hah nice catch. How come you saw this now? Did adding the type of RaidenService make the entire function mypy checkable?

Reply:
Yeah, somehow mypy started checking this.